classes/check/dirsizes.php

```diff
@@ -44,10 +43,10 @@ class dirsizes extends check {
     public function get_result(): result {
         global $CFG;
 
-        $sizedataroot = get_directory_size($CFG->dataroot);
-        $summary = $sizedataroot;
+        $sizedataroot = $this->dirsize('dataroot');
+        $summary = $this->get_directory_size_totara($CFG->dataroot);
```

Comment:
I'm a bit confused by this line.  Can you please explain why we call it here?

Reply:
Thanks. That makes sense now, but still confusing as we use now totara specific method where we don't potentially need.  

I'd suggest to change it as flowing:

 1. Introduce a new parameter to dirsize method that would define if you return size with a summary like <br>\$CFG->{$cfg} = " . display_size($size); or just a number. 
 2. Use dirsize with this new parameter. 

This way your dirsize method will return number just a number when required and it already has a logic to use a custom method when it's totara. IMHO it's cleaner way.